lib/openssl/x509.rb

```diff
@@ -99,7 +99,7 @@ def subject_key_identifier
 
           ski_asn1 = ASN1.decode(ext.value_der)
           if ext.critical? || ski_asn1.tag_class != :UNIVERSAL || ski_asn1.tag != ASN1::OCTET_STRING
-            raise ASN1::ASN1Error "invalid extension"
+            raise ASN1::ASN1Error.new("invalid extension")
```

Comment:
Can you change this to `raise ASN1::ASN1Error, "message"` - it's a little bit more idiomatic than `.new("message")`.